modules/storage-persistent-storage-pv.adoc

```diff
@@ -113,48 +129,34 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 |The volume can be mounted as read-write by many nodes.
 endif::[]
 |===
+--
+1. ReadWriteOncePod access mode for persistent volumes is a Technology Preview feature.
+--
+:FeatureName: ReadWriteOnlyPod access node for persistent volumes
```

Comment:
Couple of typo, need to update 'ReadWriteOnlyPod' to 'ReadWriteOncePod' and 'node' to 'mode'